languages/common-lisp/exercises/concept/arithmetic/.docs/introduction.md

```diff
@@ -0,0 +1,51 @@
+## Numerical Types
+
+In Common Lisp, like many languages, numbers come in a couple of types. This
+exercise focuses on getting to grips with two of the most basic number-types:
+
+- Integers: whole numbers without a decimal point (like `-6`, `0`, `25`, `1234`,
+  etc.)
+- Floating-Point Numbers: fractional or whole numbers including a decimal point (like
+  `3.14`, `-1.7`, `99.99`, `2048.0`)
+
+In general, if you are working with only whole numbers, you should prefer
+integers as they don't suffer from the same loss of precision as floating-point
+numbers do over many calculations.
+
+## Arithmetic & Prefix Notation
+
+Common Lisp uses the standard arithmetic operators for most operations (you can
```

Comment:
The [docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd) state:

> Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.

Maybe remove the link to the arithmetic operators? It could be in the `hints.md` and/or `after.md` instead.